src/main/java/de/hs_mannheim/informatik/ct/util/DocxTemplate.java

```diff
@@ -198,8 +204,10 @@ private String complexReplaceAll(String template, Pattern pattern, Function<Matc
         return template;
     }
 
-    private Iterable<Indexed<PageData<T>>> ZipPageData(Iterable<T> data, Iterable<String> qrImageIds) {
-        val dataIterator = data.iterator();
+    private Iterable<Indexed<PageData<T>>> ZipPageData(T data, Iterable<String> qrImageIds) {
```

Comment:
Hmm this method should just return `PageData<T>` since everything else only makes sense with multiple pages.